app/lib/account/backend.dart

```diff
@@ -37,17 +37,17 @@ void registerAuthenticatedUser(User user) =>
     ss.register(#_authenticated_user, user);
 
 /// The active authenticated user.
-User get authenticatedUser => ss.lookup(#_authenticated_user) as User;
+User get _authenticatedUser => ss.lookup(#_authenticated_user) as User;
```

Comment:
Fantastic initiative..

Maybe we should consider moving `_userAuthWrapper` or part of it into `backend.dart`, that way `registerAuthenticatedUser` can also be hidden...
I know we might need `registerAuthenticatedUser` in tests, but then we could annotated it `@visibleForTesting`

Reply:
Note: not in this PR :)